pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Display a list of hosts belonging to a group:
+
+`ansible {{group}} --list-hosts`
+
+- Ping a group of hosts:
+
+`ansible {{group}} --module-name ping`
+
+- Display facts about a group of hosts:
+
+`ansible {{group}} --module-name setup`
+
+- Execute a command (using the command module) on a group of hosts:
+
+`ansible {{group}} --module-name command --args '{{my command}}'`
+
+- Execute a command with administrative priviledges:
+
+`ansible {{group}} --become --ask-become-pass --module-name command --args '{{my command}}'`
+
+- Execute a command using a custom inventory file:
+
+`ansible {{group}} --inventory-file {{path/to/inventory/file}} --module-name command --args '{{my command}}'`
```

Comment:
If by using descriptions we can hint at what each single-letter option stands for, then perhaps long-form options may be unnecessary and we can keep the commands succinct. Would the following pass muster?

- List hosts belonging to a group:

`ansible {{group}} --list-hosts`

- Ping a group of hosts by invoking the ping module:

`ansible {{group}} -m ping`

- Display facts about a group of hosts by invoking the setup module:

`ansible {{group}} -m setup`

- Execute a command on a group of hosts by invoking command module with arguments:

`ansible {{group}} -m command -a '{{my command}}'`

- Execute a command with administrative privileges:

`ansible {{group}} --become --ask-become-pass -m command -a '{{my command}}'`

- Execute a command using a custom inventory file:

`ansible {{group}} -i {{inventory_file}} -m command -a '{{my command}}'`